src/libstd/io/mod.rs

```diff
@@ -332,6 +597,17 @@ pub trait Write {
     ///
     /// It is considered an error if not all bytes could be written due to
     /// I/O errors or EOF being reached.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use std::io::Write;
+    ///
+    /// let mut buffer = Vec::new();
```

Comment:
This example may want to use a `BufWriter` because flushing on vectors or files doesn't actually do anything.